ccmlib/scylla_node.py

```diff
@@ -1127,13 +1127,12 @@ def update_yaml(self):
                 self.get_base_cassandra_version() >= 1.2):
             _, data['native_transport_port'] = self.network_interfaces['binary']
 
-        data['data_file_directories'] = [os.path.join(self.get_path(), 'data')]
+        data['workdir,W'] = self.get_path()
```

Comment:
`'workdir,W'` is this a typo ? what kind of a configuration key is that ?

Reply:
"workdir,W" is used instead of "workdir", because scylla defines this option this way and dtests compare the names of used options with the names defined in scylla.